.taskcluster.yml

```diff
@@ -2,19 +2,22 @@
 version: 1
 reporting: checks-v1
 policy:
-    pullRequests: public
+    pullRequests: collaborators
 tasks:
     - $let:
           taskgraph:
               branch: taskgraph
-              revision: c78d65237c6ed064b9cbb3cd063b22e9e052a06a
+              revision: 7dde9ce7740068d7b73218976343a28fc99be847
           template:
-              repo: https://github.com/mozilla-extensions/xpi-template
-              branch: master
+              #XXX uncomment to merge
+              #repo: https://github.com/mozilla-extensions/xpi-template
+              #branch: master
+              repo: https://github.com/escapewindow/xpi-template
+              branch: private-ci
           trustDomain: xpi
           # XXX for private repos, uncomment this line, and comment out the next
-          # githubCloneSecret: project/xpi/xpi-github-clone-ssh
-          githubCloneSecret: ""
+          #xpiSshSecretName: project/xpi/xpi-github-clone-ssh
+          xpiSshSecretName: ""
           # XXX use "privileged" or "system" to enable signing
           xpiSigningType: ""
```

Comment:
Thinking about this a bit, I wonder if it would make sense to move these settings to the top (potentially even to their own out let block), since they are the only thing users will need to change.

That is the same as the motivation for having this let block; the ideal for that is that people using taskgraph would only ever need to touch this let block, and the rest would be boiler plate. That hasn't happened in practice, but I suspect much of the variation could be abstracted with a bit of care.

Reply:
Moved to the top of the let block. And yeah, I'm hoping that's true, which may mean we can have .travis.yml sized .taskcluster.yml's.